notebooks/121-convert-to-openvino/121-convert-to-openvino.ipynb

```diff
@@ -0,0 +1,1649 @@
+      "\t\t\tused with TensorBoard.\n",
+      "  --tensorflow_custom_layer_libraries \n",
+      "\t\t\tTensorFlow*: comma separated list of shared libraries with TensorFlow*\n",
+      "\t\t\tcustom operations implementation.\n",
+      "\n",
+      "MXNet-specific parameters:\n",
+      "  --input_symbol \n",
+      "\t\t\tSymbol file (for example, model-symbol.json) that contains a topology\n",
+      "\t\t\tstructure and layer attributes\n",
+      "  --nd_prefix_name \n",
+      "\t\t\tPrefix name for args.nd and argx.nd files.\n",
+      "  --pretrained_model_name \n",
+      "\t\t\tName of a pretrained MXNet model without extension and epoch number.\n",
+      "\t\t\tThis model will be merged with args.nd and argx.nd files\n",
+      "  --save_params_from_nd \n",
+      "\t\t\tEnable saving built parameters file from .nd files\n",
+      "  --legacy_mxnet_model \n",
+      "\t\t\tEnable MXNet loader to make a model compatible with the latest MXNet\n",
+      "\t\t\tversion. Use only if your model was trained with MXNet version lower\n",
+      "\t\t\tthan 1.0.0\n",
+      "  --enable_ssd_gluoncv \n",
+      "\t\t\tEnable pattern matchers replacers for converting gluoncv ssd topologies.\n",
+      "\t\t\t\n",
+      "\n",
+      "Caffe*-specific parameters:\n",
+      "  --input_proto \n",
+      "\t\t\tDeploy-ready prototxt file that contains a topology structure and\n",
+      "\t\t\tlayer attributes\n",
+      "  --caffe_parser_path \n",
+      "\t\t\tPath to Python Caffe* parser generated from caffe.proto\n",
+      "  --k \n",
+      "\t\t\tPath to CustomLayersMapping.xml to register custom layers\n",
+      "  --disable_omitting_optional \n",
+      "\t\t\tDisable omitting optional attributes to be used for custom layers.\n",
+      "\t\t\tUse this option if you want to transfer all attributes of a custom layer\n",
+      "\t\t\tto IR. Default behavior is to transfer the attributes with default values\n",
+      "\t\t\tand the attributes defined by the user to IR.\n",
+      "  --enable_flattening_nested_params \n",
+      "\t\t\tEnable flattening optional params to be used for custom layers. Use\n",
+      "\t\t\tthis option if you want to transfer attributes of a custom layer to IR\n",
+      "\t\t\twith flattened nested parameters. Default behavior is to transfer\n",
+      "\t\t\tthe attributes without flattening nested parameters.\n",
+      "\n",
+      "Kaldi-specific parameters:\n",
+      "  --counts \n",
+      "\t\t\tPath to the counts file\n",
+      "  --remove_output_softmax \n",
+      "\t\t\tRemoves the SoftMax layer that is the output layer\n",
+      "  --remove_memory \n",
+      "\t\t\tRemoves the Memory layer and use additional inputs outputs instead\n",
+      "\t\t\t\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Python conversion API parameters description\n",
+    "from openvino.tools.mo import convert_model\n",
```

Comment:
`from openvino.tools.mo import convert_model` 
In most notebooks, an import of mo used, not only one method. I suggest keep that approach:
`from openvino.tools import mo` 
`mo.convert_model(help=True)`

Reply:
Imports updated